client/src/components/Options.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {
+  ColoredText, OptContainer, OptionsBox, OptionsIcon, IconWrapper, ModuleWrapper,
+} from './styles/styles';
+
+const Options = ({ options }) => (
+  <div className="options">
+    <ModuleWrapper>
+      {options[0] && (
+        <OptContainer>
```

Comment:
Nice use of the && to render the component - really clean. Might also be clean to create a smaller options component in its own file, import it here, and then map over the options array and for each, render that smaller options component while passing down the option name/image? Overall works just fine whether you do it that way or this way